core/src/main/java/com/scalar/db/api/DistributedTransactionManager.java

```diff
@@ -56,9 +56,11 @@ public interface DistributedTransactionManager {
    * Begins a new transaction.
    *
    * @return {@link DistributedTransaction}
+   * @throws TransactionNotFoundException if beginning the transaction fails, but it's retryable.
```

Comment:
```suggestion
   * @throws TransactionNotFoundException if the transaction fails to start.
```
I removed `but it's retryable` because I feel it's a bit redundant with the next sentence.
Ditto for the others.

Reply:
I think removing `but it's retryable` may make the statement sound unsure about the difference from the `TransactionException`. So, how about removing the next sentence?

Reply:
OK, then, how about something like this to make things clear?
```
@throws TransactionNotFoundException if the transaction fails to start due to xxx. You can retry the transaction.
@throws TransactionException if the transaction fails to start due to yyy. You cannot retry the transaction.
```

Can we say something for `xxx` and `yyy`?

Reply:
Thank you for the suggestion.

Actually, it's hard to say concrete reasons for `TransactionNotFoundException` because it's only occurs in ScalarDB Cluster.

So, how about some thing like the following?

```
@throws TransactionNotFoundException if the transaction fails to start due to retryable reasons. You can retry the transaction.
@throws TransactionException if the transaction fails to start.
```

Reply:
How about this?
```
@throws TransactionNotFoundException if the transaction fails to start due to transient faults. You can retry the transaction.
@throws TransactionException if the transaction fails to start due to transient or nontransient faults. You can try retrying the transaction, but you may not be able to start the transaction due to nontransient faults.
```

Reply:
Thank you for the suggestion! Fixed in 59d84339872f8c1859e78f051240c317cde53c55. Please take a look when you have time!